tests/common/query_context_generator.py

```diff
@@ -253,6 +253,7 @@ def generate(
                 )
             ],
             "result_type": ChartDataResultType.FULL,
+            "form_data": {},
```

Comment:
`form_data` should probably be added to the signature of the method and passed as-is to here instead of defaulting to an empty dict.